vo_models/vodataservice/models.py

```diff
@@ -246,7 +246,7 @@ def validate_lists(cls, value):
         return value
 
 
-class TableSchema(BaseXmlModel, tag="schema", ns="", skip_empty=True):
+class TableSchema(BaseXmlModel, tag="schema", ns="", nsmap={"": ""}, skip_empty=True):
```

Comment:
Needed to force pydantic-xml to not qualify the `TableSchema` and `Table` elements (and their children) when included as part of a VOSITables model.